components/classroom/main.js

```diff
@@ -3,31 +3,41 @@
 import { getProjectIDFromURL } from './groups/utils/project.js';
 import Roles from "./roles.mjs";
 import hasPermission from "./hasPermission";
-import { Action, Scope, Entity } from './permissions_parameters.mjs';
+import TPEN from '../../TPEN/index.mjs';
 
 
 // Function to fetch project data from the TPEN API
 async function fetchProjectData(projectId) {
```

Comment:
Instead of putting the fetchProjectData function inside main, I think it would be better if it stayed inside fetchData.mjs just so it can be exported to other parts of the code for more reusability